### ISSUE TITLE
about gradient backward

### ISSUE THREAD

**Issue by www516717402** [jiang-du/BlazePose-tensorflow#9]
Thank you for open this project. 
1. In paper ** The gradients from the re- gression encoder are not propagated back to the heatmap- trained features (note the gradient-stopping connections in Figure 4).** However, not see this operate in your train step.
2. Offset also mention in paper.
Look forward your reply.

**Comment by www516717402** [comment]
thank you, I know how to build this model in pytorch

**Comment by TYZQ** [comment]
> 谢谢，我知道如何在pytorch中制作这个模型

Can you share me your pytorch version?
